cmd/network/verification.go

```diff
@@ -0,0 +1,101 @@
+package network
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	ocmlog "github.com/openshift-online/ocm-sdk-go/logging"
+	"github.com/openshift/osd-network-verifier/pkg/cloudclient"
+	"github.com/spf13/cobra"
+)
+
+var (
+	defaultTags            = map[string]string{"osd-network-verifier": "owned", "red-hat-managed": "true", "Name": "osd-network-verifier"}
```

Comment:
I don't think this as needed, as the verifier already uses these as the default tags

Reply:
The defaults in verifier also come from the command, the cloud client itself doesn't have defaults.